go.mod

```diff
@@ -16,7 +16,7 @@ require (
 	k8s.io/client-go v10.0.0+incompatible
 	k8s.io/klog v1.0.0
 	k8s.io/kube-openapi v0.0.0-20200410163147-594e756bea31
-	k8s.io/sample-controller v0.0.0-00010101000000-000000000000
+	k8s.io/sample-controller v0.16.15
```

Comment:
Why do we need sampler controller?

Reply:
Is there a way to get rid of it? Feels a little weird to rely on it.